src/main/java/fi/aalto/cs/apluscourses/intellij/actions/CourseProjectAction.java

```diff
@@ -73,13 +77,15 @@ public CourseProjectAction(@NotNull CourseFactory courseFactory,
                              @NotNull SettingsImporter settingsImporter,
                              @NotNull ComponentInstaller.Factory installerFactory,
                              @NotNull PostponedRunnable ideRestarter,
-                             @NotNull CourseProjectActionDialogs dialogs) {
+                             @NotNull CourseProjectActionDialogs dialogs,
+                             @NotNull InstallerDialogs.Factory installerDialogsFactory) {
     this.courseFactory = courseFactory;
     this.createCourseFile = createCourseFile;
     this.settingsImporter = settingsImporter;
     this.installerFactory = installerFactory;
     this.ideRestarter = ideRestarter;
     this.dialogs = dialogs;
+    this.installerDialogsFactory = installerDialogsFactory;
```

Comment:
This isn't set in the constructor with no parameters, which causes a NPE when using this action